src/components/Onboarding.jsx

```diff
@@ -0,0 +1,144 @@
+import * as rb from 'react-bootstrap'
+import React, { useState } from 'react'
+import Sprite from './Sprite'
+import { useSettingsDispatch } from '../context/SettingsContext'
+
+export default function Onboarding() {
+  const [step, setStep] = useState(0)
+  const settingsDispatch = useSettingsDispatch()
+
+  if (step === 0) {
+    return (
+      <div class="mx-auto my-auto py-5">
+        <div class="d-flex justify-content-center">
+          <Sprite symbol="logo" width="200" height="200" className="mb-4" />
+        </div>
+        <h1 class="text-center">JoinMarket</h1>
+        <p class="text-center fs-4 text-secondary mb-5">Top-notch privacy for your bitcoin.</p>
+        <div class="d-flex justify-content-center mb-2">
+          <rb.Button size="lg" className="col-10" variant="dark" onClick={() => setStep(1)}>
+            Get started
+          </rb.Button>
+        </div>
+        <div class="d-flex justify-content-center">
+          <rb.Button variant="dark" onClick={() => settingsDispatch({ showOnboarding: false })}>
+            Skip intro
+          </rb.Button>
+        </div>
+        <p class="mt-4 text-center">
+          Your wallet, your coins. <br />
+          100% open-source & open-design.
+        </p>
+      </div>
+    )
+  } else if (step === 1) {
+    return (
+      <div class="mx-auto my-auto py-5 col-10 col-md-4">
+        <div class="d-flex justify-content-center">
+          <Sprite symbol="welcome" width="150" height="150" className="mb-4" />
+        </div>
+        <p class="fs-3 text-center">1: Welcome to JoinMarket!</p>
```

Comment:
```suggestion
        <p class="fs-3 text-center">Welcome to JoinMarket!</p>
```

I'd drop the numbering here. Or alternatively do something like 1/5 below or so.

Reply:
Ah, yes, my mistake. I numbered the headlines in the text to make it clear, but I agree, we should remove them in the real thing.